pycaret/internal/plots/time_series.py

```diff
@@ -31,17 +31,17 @@ def plot_(
 ) -> Optional[Any]:
     if plot == "ts":
         plot_data = plot_series(data=data, return_data=return_data, show=show)
-    elif plot == "splits-tt":
+    elif plot == "train_test_split":
         plot_data = plot_splits_tt(
             train=train, test=test, return_data=return_data, show=show
         )
-    elif plot == "splits_cv":
-        plot_data = plot_splits_cv(data=data, cv=cv, return_data=return_data, show=show)
+    elif plot == "cv":
+        plot_data = plot_cv(data=data, cv=cv, return_data=return_data, show=show)
     elif plot == "acf":
         plot_data = plot_acf(data=data, return_data=return_data, show=show)
     elif plot == "pacf":
         plot_data = plot_pacf(data=data, return_data=return_data, show=show)
-    elif plot == "predictions":
+    elif plot == "forecast":
```

Comment:
Is this somewhere in the documentation? If so, has it been updated?

Reply:
@moezali1  will update the documentation before the next release.